executor/services/workspace_service.py

```
@@ -0,0 +1,263 @@
+    """
+    Get list of Git-tracked files in workspace.
+
+    Args:
+        workspace_path: Path to the workspace
+
+    Returns:
+        List of relative file paths
+    """
+    try:
+        result = subprocess.run(
+            ["git", "ls-files"],
```

Ruff (0.14.14)

[error] Starting a process with a partial executable path

(S607)